InfoMovies/InfoMovies/Converters/AspectRatioConverter.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace InfoMovies.Converters
+{
+    public class AspectRatioConverter : IValueConverter
+    {
+        #region Properties
+
+        public float Ratio { set; get; }
+
+        #endregion
+
+        #region IValueConverter
+
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
```

Comment:
How could you improve this Converter so it becomes more secure? (Less error prone)

Reply:
validating the `Ratio` parameter and creating exceptions to notify developer (like `DivideByZeroException` and `NullParameter`)